autoload/flood/check.vim

```diff
@@ -94,10 +129,20 @@ function! flood#check#run(...) abort
   endif
 
   let mode = a:0 > 0 ? a:1 : 'r'
-  let cmd = printf('%s --json', flood#flowbin())
-  let s:job = job_start(cmd, {
-        \ 'callback': {c, m -> s:check_callback(c, m, mode)},
-        \ })
+  let cmd = [flood#flowbin(), '--json']
+
+  if has('nvim')
```

Comment:
I think we can create `job executer` function at flood.vim

```viml
function! flood#job_executer(cmd, args, callback_handlers)
  if has('nvim')
    " nvim job start
  else
    " vim job start
  endif
endif
```

all `flood/*.vim`'s job_start should call `flood#job_executer`.

What do you think?